src/main/java/org/springframework/integration/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -207,27 +203,29 @@ public Executor getFetchTaskExecutor() {
 
 	/**
 	 * The task executor for fetch operations
+	 *
 	 * @param fetchTaskExecutor the Executor for fetch operations
 	 */
 	public void setFetchTaskExecutor(Executor fetchTaskExecutor) {
 		this.fetchTaskExecutor = fetchTaskExecutor;
 	}
 
-
 	public Executor getAdminTaskExecutor() {
 		return adminTaskExecutor;
 	}
 
 	/**
-	 * The task executor for leader and offset updates
-	 * @param adminTaskExecutor the task executor for leader and offset updates.
+	 * The task executor for leader, offset, and partition reassignment updates
+	 *
+	 * @param adminTaskScheduler the task executor for leader, offset and partition reassignment updates
 	 */
-	public void setAdminTaskExecutor(Executor adminTaskExecutor) {
-		this.adminTaskExecutor = adminTaskExecutor;
+	public void setAdminTaskExecutor(Executor adminTaskScheduler) {
```

Comment:
Haven't you missed anything here?
Otherwise what is the reason to rename it to the `Scheduler`...